Themes/default/languages/Help.english.php

```diff
@@ -252,6 +252,7 @@
 $helptxt['enableCompressedOutput'] = 'This setting will compress output in order to lower bandwidth consumption, but it requires
 		zlib to be installed.';
 $helptxt['httponlyCookies'] = 'Cookies won\'t be accessible by scripting languages, such as JavaScript. This setting can help to reduce identity theft through XSS attacks. This can cause issues with third party scripts but should be on wherever possible.';
+$helptxt['samesiteCookies'] = 'The SameSite attribute of the Set-Cookie HTTP response header allows you to declare if your cookie should be restricted to a first-party or same-site context. For further information, please refer to <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite">MDN</a>';
```

Comment:
Just a note, and I may be guilty of this as well, the "en-US/" could be omitted from the URL, it looks like Mozilla redirects to the correct website via locale checks they do.  Although our translations can also link to a specific documentation as well here, so it isn't a big worry.  But since english is the fallback, it may produce an undesired link.  For this specific link, they only have a few translations.

Reply:
Should I change this?  I'm fine either way....

Reply:
I'm not sure.  Maybe one of our translators can weigh in on this.  I see it kinda like php.net links.  I try not to link to the "manual/en" in the urls, as they will detect a locale and direct to a translated page automatically.  I kinda actually blame them for not having a "link to this page" type url which provides the most optimized url of that those linking to page can use (getting rid of any junk in the url).

Reply:
It really does make sense to get rid of the hardcoded language pref.  Done.